data/human/free worlds outfits.txt

```diff
@@ -34,4 +34,4 @@ outfit "F215-BU Shield Generator"
 	"outfit space" -180
 	"shield generation" 4.53
 	"shield energy" 4.53
-	description "This huge shield generator is manufactured almost exclusively for the Skein, but as it isn't based on any more advanced technology than a standard civilian shield, it is left unlicensed so any willing pilot can squeeze one into their ship."
+	description "This huge shield generator is manufactured almost exclusively for the Skein, but as it isn't based on any more advanced technology than a standard civilian shield, the Free Worlds has left it unlicensed so any willing pilot can squeeze one into their ship."
```

Comment:
"the Free Worlds has" doesn't sound right to me.

Reply:
We are treating "Free Worlds" as a singular entity. Nobody really says "the United States have..." because the individual states themselves didn't do the thing collectively, but rather the entire entity that represents the (_the United States_).

Reply:
Ok, based on a precedent I found in the Skein's description, I guess you're right.